sites/fast-website/src/app/components/side-navigation/side-navigation.template.ts

```diff
@@ -14,6 +14,7 @@ export const SideNavigationTemplate = html<SideNavigation>`
                                 href=${x => x.actionLink}
                                 appearance="lightweight"
                                 :innerHTML=${x => x.icon}
+                                aria-label=${x => x.header}
```

Comment:
I think you'll want a specific property for this unless "Header" is specifically set to show "Medium", "Github", etc...

Reply:
switching to use `actionText` instead of `header`